docs/contents/index.md

```diff
@@ -118,5 +118,5 @@ Machine Images), but it does use the EKS Optimized AMI. See the project
 repository for the [EKS Optimized AMI](https://github.com/awslabs/amazon-eks-ami)
 if you are interested in the AL2 container runtime kernel version.
 
```

Comment:
Good catch to add these, but somewhere it seems like we should have a complete list of releases including the old. Should this get broken down to a 1-18 and 1-19 section?

Reply:
Latest first

Reply:
I created a ticket to enumerate all these various links in the runbook. We should have a ticket for each one so we don't miss them or some sort of checkbox

Reply:
> Good catch to add these, but somewhere it seems like we should have a complete list of releases including the old. Should this get broken down to a 1-18 and 1-19 section?

I see what you mean. Do you want me to make that change? I think this is one of those things that ends up on the website. Do we want all the releases on there?

Reply:
Yes. I have no better idea right now, it seems like all the old releases should be listed somewhere.